Common.mk

```diff
@@ -149,8 +149,11 @@ IMAGE_IMPORT_CACHE?=type=registry,ref=$(LATEST_IMAGE) type=registry,ref=$(subst
 
 BUILD_OCI_TARS?=false
 
-LOCAL_IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(image)/images/amd64) $(if $(filter true,$(HELM_PULL)),helm/pull,) $(if $(filter true,$(HAS_HELM_CHART)),helm/build,) 
-IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(if $(filter true,$(BUILD_OCI_TARS)),$(call IMAGE_TARGETS_FOR_NAME,$(image)),$(image)/images/push)) $(if $(filter true,$(HAS_HELM_CHART)),helm/push,) 
+LOCAL_IMAGE_WO_HELM_TARGETS=$(foreach image,$(IMAGE_NAMES),$(image)/images/amd64)
```

Comment:
Do we need these at all? Were you thinking of preserving the make images behavior for people who might run it locally?

Reply:
Yes, I'm hoping to keep `make images` the same as before as I know there are quite bit of people use this target often for testing. I'm also open to change the default behavior, but would like to hear what is the best way to proceed.

Reply:
honestly im fine changing it. we can just tell people in the build channel and they can adjust :)

Reply:
Changed :)